index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hoja de Vida</title>
+  <link rel="stylesheet" href="Styles.css">
+</head>
+<body>
+<div class="sidebar">
+  <div class="circle-container">
+    <div class="circle-large">
+      <div class="circle-outline"></div>
+      <div class="circle-small"></div>
+    </div>
+  </div>
+
+  <h3>Contacto</h3>
+  <p><strong>Email</strong>: guevararodriguezsofia@example.com<br><strong>Teléfono:</strong><br>+57 300 844223<br>Bogotá, Colombia</p>
```

Comment:
```suggestion
  <p><strong>Email</strong>: guevararodriguezsofia@example.com</p>
  <p><strong>Teléfono:</strong>+57 300 844223 </p>
  <p>Bogotá, Colombia</p>
```

Para obtener el efecto de verticalidad basta con volver cada unidad un párrafo